receiver/hostmetricsreceiver/factory.go

```diff
@@ -123,6 +131,13 @@ func createAddScraperOptions(
 		return nil, fmt.Errorf("host metrics scraper factory not found for key: %q", key)
 	}
 
+	if !featuregate.GetRegistry().IsEnabled(internal.EmitMetricsWithDirectionAttributeFeatureGateID) {
+		logDeprecatedFeatureGateForDirection(set.Logger, internal.EmitMetricsWithDirectionAttributeFeatureGateID)
+	}
```

Comment:
Do we need warning on the `EmitMetricsWithDirectionAttributeFeatureGateID` feature gates given that they are enabled by default anyway? Maye just keep it only for `EmitMetricsWithoutDirectionAttributeFeatureGateID`?

Reply:
I was thinking it would be worth emitting a log in case someone out there is using these to disable metrics w/ direction. Upon upgrade to 0.62.0, when the gates are removed, the collector would fail to start because the gate is no longer registered.

Reply:
Ok makes sense. Let's keep them